raml-parser-2/src/main/java/org/raml/v2/internal/impl/commons/model/type/TypeDeclaration.java

```diff
@@ -308,6 +310,23 @@ public String internalFragment()
         }
     }
 
+    public String schemaPath()
+    {
+        ResolvedType resolvedType = getResolvedType();
+        if (resolvedType instanceof AbstractExternalType)
+        {
+            if (!((AbstractExternalType) resolvedType).getSchemaPath().equalsIgnoreCase(ARTIFICIAL_NODE))
```

Comment:
Why did you add this check?

Reply:
No reason, I can remove it if you want.
I thought it was not right to expose that string to the user